src/vm/i386/asmconstants.h

```diff
@@ -467,6 +467,31 @@ ASMCONSTANTS_C_ASSERT(Thread__m_pDomain == offsetof(Thread, m_pDomain));
 
 #endif
 
+#if !defined(CROSSGEN_COMPILE) && !defined(DACCESS_COMPILE)
+// ResolveCacheElem from src/vm/virtualcallstub.h
+#define ResolveCacheElem__sizeof_mt                 4
+#define ResolveCacheElem__sizeof_token              4
+#define ResolveCacheElem__sizeof_target             4
+#define ResolveCacheElem__sizeof_indirect_addr      4
+#define ResolveCacheElem__sizeof_caller_ret_addr    4
+
+#define ResolveCacheElem__offset_mt                 0
+#define ResolveCacheElem__offset_token              (ResolveCacheElem__offset_mt + ResolveCacheElem__sizeof_mt)
+#define ResolveCacheElem__offset_target             (ResolveCacheElem__offset_token + ResolveCacheElem__sizeof_token)
+#define ResolveCacheElem__offset_next               (ResolveCacheElem__offset_target + ResolveCacheElem__sizeof_target)
+
+ASMCONSTANTS_C_ASSERT(ResolveCacheElem__sizeof_mt               == sizeof(void *));
```

Comment:
I was thinking that you'd use convention that's consistent with the other constants in asmconstants.h both in naming and the way of defining them. Could you please do it that way?
All the existing offsets with the exception of one class are named as Class__Element, so in your names, it would be nice to drop the `_offset` from the names and also drop the "sizeof" constants and define the offsets as absolute values. That way it would be consistent with the rest.